### ISSUE TITLE
Encoding URLs?

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/mesh-viewer#17]
It seems like the mesh-viewer might be encoding the URLs returned from the server in the mesh json.

For example, the viewer here:

https://www.facebase.org/mesh-viewer/view.html?model=/ermrest/catalog/1/entity/viz:model_json/id=11

Is getting the json from:

https://www.facebase.org/ermrest/catalog/1/entity/viz:model_json/id=11

In the json, the mesh.url is `"/hatrac/facebase/previews/fb2/FB00000913/YC391CCMB_Wnt1-Cre%3BAlk5flfl_P0_Hard_Tissue.obj.gz"`. This appears to be correctly encoded.

But looking at the mesh viewer console or network monitor, it is trying to get:

`https://www.facebase.org/hatrac/facebase/previews/fb2/FB00000913/YC391CCMB_Wnt1-Cre%253BAlk5flfl_P0_Hard_Tissue.obj.gz`

The `%` in `%3B` is getting encoded again resulting in `%253B` and therefore a `404 Not Found` error.

**Comment by NickolausDS** [comment]
The URL: 

https://www.facebase.org/ermrest/catalog/1/entity/viz:model_json/id=11

Doesn't appear to point to a model anymore, I only get `[]`, but the obj.gz works. I did a quick workaround by creating a new model that points to the file: 

```
[{"RCB": null,
  "RCT": "2017-09-22T17:33:18.797126-07:00",
  "RID": 4453,
  "RMB": null,
  "RMT": "2017-09-22T17:33:18.797126-07:00",
  "bboxcolor": [1, 1, 0],
  "bgcolor": [0, 0, 0],
  "description": null,
  "id": "footest",
  "label": "Foo Test",
  "landmark": [],
  "mesh": [{"color": [0.937254901960784, 0.854901960784314, 0.184313725490196],
            "description": "This is a test that should fail due to double encoding the URL",
            "id": "footest",
            "label": "foo-test",
            "opacity": 1,
            "url": "https://dev.facebase.org/hatrac/facebase/previews/fb2/FB00000913/YC391CCMB_Wnt1-Cre%3BAlk5flfl_P0_Hard_Tissue.obj.gz"}],
  "volume": null}]
```

After that the urlencoding problem shows up normally, with `%3B`, a `;`, getting double encoded to `%253B`. I do notice that manually downloading the file produces a file with a different filename than the original URL, but that doesn't strike me as a problem.

**Comment by NickolausDS** [comment]
This issue can also be closed, referencing merged PR: #21
